### PR TITLE
Increase time, memory limit for complete_psd

### LINKED ISSUE #697: [Bug]: CORS Error on master list server
### Expected Behavior

Successfully upload psd cg file

### Actual Behavior

CORS Error during uploading psd cg file

image

https://s3.ap-northeast-2.amazonaws.com/luda-editor/master/www/index.html

### Steps to Reproduce

_No response_

### Other Info

_No response_

### PR DESCRIPTION
**Pull request by bigfoodK**
https://github.com/NamseEnt/namseent/issues/697#issuecomment-1787408716
Closes #697

### CODE REVIEW

File: luda-editor/cdk/lib/cdk-stack.ts [hunk 1]
```diff
@@ -110,6 +110,8 @@ export class CdkStack extends cdk.Stack {
                     ],
                 }),
             ],
+            timeout: cdk.Duration.seconds(30),
+            memorySize: 512,
```
Comment (namse) [inline luda-editor/cdk/lib/cdk-stack.ts:114]: How much time taken for this?
Increasing memory will get more cpu too so it would be faster and effective, economic.
Benchmark is the best way to optimize, but optimization is not a essential right now.
I just want the information how much it takes with current setting.

**Review by namse — APPROVED** [review]

File: luda-editor/cdk/lib/cdk-stack.ts (hunk 1 above)
Comment (bigfoodK) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: > It seems to be problem that occurs because `complete_psd` is too heavy a task. In lambda log, runtime exited because of timeout. after increase timeout and memory size, It worked. for [오하연-놀람혹은히익-사복(test).psd](https://drive.google.com/file/d/112GoZO9L25aKv8OPJ_DU10_BvKDO1ymN/view?usp=drive_link), It tooks 6684ms and 266MB.

It tooks 6684ms and 266MB for [오하연-놀람혹은히익-사복(test).psd](https://drive.google.com/file/d/112GoZO9L25aKv8OPJ_DU10_BvKDO1ymN/view?usp=drive_link)

Comment (namse) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: no test with 512?

Comment (bigfoodK) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: That is result of 30sec and 512M limits.

Comment (namse) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: You mean it made a timeout with 512m memory limit?

Comment (namse) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: FYI
https://docs.aws.amazon.com/lambda/latest/operatorguide/computing-power.html
image

Comment (bigfoodK) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: No, I set time limit to 30sec, and set memory limit to 512m.
Then it worked and result was
> It tooks 6684ms and 266MB for [오하연-놀람혹은히익-사복(test).psd](https://drive.google.com/file/d/112GoZO9L25aKv8OPJ_DU10_BvKDO1ymN/view?usp=drive_link)

Comment (namse) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: with 512MB, it's 6sec, right?
Test with 4 cores or 6 cores. it would be cheaper than 1 core with 512MB.

Comment (namse) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: I never used this tool but I know it's very super powerful to find best memory settting for lambda  https://github.com/alexcasalboni/aws-lambda-power-tuning

Comment (bigfoodK) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: [Result of 512~3008](https://lambda-power-tuning.show/#AAIVAyoEQAVVBmoHqgrACw==;VcG1RatydkWrgjFFq6IURQCg60Sr+tJEALDLRFWVwkQ=;CPJMOP0hVjhlZlA4OQVcOGxDUjjPdlw45BqZOA8koTg=)

I can't test 3008+ due to quota. 
> New AWS accounts have reduced concurrency and memory quotas. AWS raises these quotas automatically based on your usage.

https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html

Comment (bigfoodK) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: Additionally, I tested a light api, `list_project_sequences`.
In this case, lower memory is better. [result](https://lambda-power-tuning.show/#AAIVAyoEQAVVBmoHqgrACw==;q6qWQauqhkEAAJBBVVU1QauqWkFVVYVBq6pSQauqUkE=;hF4rNNFIbDQ2Aqk0WQ6ONL7jxzSfGQ41VVIoNUl2OTU=)

However, compared to `complete_psd`, the impact does not seem significant. [compare](https://lambda-power-tuning.show/#AAIVAyoEQAVVBmoHqgrACw==;VcG1RatydkWrgjFFq6IURQCg60Sr+tJEALDLRFWVwkQ=;CPJMOP0hVjhlZlA4OQVcOGxDUjjPdlw45BqZOA8koTg=;AAIVAyoEQAVVBmoHqgrACw==;q6qWQauqhkEAAJBBVVU1QauqWkFVVYVBq6pSQauqUkE=;hF4rNNFIbDQ2Aqk0WQ6ONL7jxzSfGQ41VVIoNUl2OTU=;complete_psd;list_project_sequences
).


In conclusion, I think it is good to set the memory size to 1621.

Comment (namse) [reply luda-editor/cdk/lib/cdk-stack.ts:114]: They said there are cpu ceiling for lambda with memory limit https://stackoverflow.com/a/66523153
hmm. OK. I confirm 1621 memory size
